### PR TITLE
Feature/aliases in api 482

### LINKED ISSUE #482: Aliases are not resolved in column matching
Aliases in from-clauses will cause an error when parsing column elements. There must be a structure that keeps track of aliases while parsing to resolve them appropriately.

### PR DESCRIPTION
**Pull request by MGlauer**
Closes #482

### CODE REVIEW

File: api/tests/test_regression/test_issue_482.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import json
+
+import requests
+
+from api.tests import APITestCase
+
+from ..util import load_content_as_json
+
+
+class Test482(APITestCase):
```
Comment (Bachibouzouk) [inline api/tests/test_regression/test_issue_482.py]: I would be more in favor of an explicit name to make clear what is tested here without the need to look through the issues
```suggestion
class TestAliasesTracking(APITestCase):
```

File: api/tests/test_regression/test_issue_482.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+        resp = self.__class__.client.put(
+            "/api/v0/schema/{schema}/tables/{table}/".format(
+                schema=self.test_schema, table=self.test_table
+            ),
+            data=json.dumps({"query": self._structure_data}),
+            HTTP_AUTHORIZATION="Token %s" % self.__class__.token,
+            content_type="application/json",
+        )
+
```
Comment (Bachibouzouk) [inline api/tests/test_regression/test_issue_482.py]: A comment about what the assertion is about in plain language? (no need to be super long)

File: api/tests/test_regression/test_issue_482.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        resp = self.__class__.client.post(
+            "/api/v0/schema/{schema}/tables/{table}/rows/new".format(
+                schema=self.test_schema, table=self.test_table
+            ),
+            data=json.dumps({"query": [{"name": "Hans"}, {"name": "Petra"}]}),
+            HTTP_AUTHORIZATION="Token %s" % self.__class__.token,
+            content_type="application/json",
+        )
+
```
Comment (Bachibouzouk) [inline api/tests/test_regression/test_issue_482.py]: comment?

File: api/tests/test_regression/test_issue_482.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        self.assertEqual(
+            resp.status_code,
+            201,
+            load_content_as_json(resp).get("reason", "No reason returned"),
+        )
+
+    def test_482(self):
```
Comment (Bachibouzouk) [inline api/tests/test_regression/test_issue_482.py]:
```suggestion
    def test_aliases_in_form_clauses(self):
```

File: api/tests/test_regression/test_issue_482.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+    def tearDown(self):
+        resp = self.__class__.client.delete(
+            "/api/v0/schema/{schema}/tables/{table}/".format(
+                schema=self.test_schema, table=self.test_table
+            ),
+            HTTP_AUTHORIZATION="Token %s" % self.__class__.token,
+            content_type="application/json",
+        )
+
```
Comment (Bachibouzouk) [inline api/tests/test_regression/test_issue_482.py]: comment?

File: api/tests/test_regression/test_issue_482.py (hunk 2 above)
Comment (MGlauer) [reply api/tests/test_regression/test_issue_482.py]: Done in 592f8c7

File: api/tests/test_regression/test_issue_482.py (hunk 3 above)
Comment (MGlauer) [reply api/tests/test_regression/test_issue_482.py]: Done in 592f8c7

File: api/tests/test_regression/test_issue_482.py (hunk 5 above)
Comment (MGlauer) [reply api/tests/test_regression/test_issue_482.py]: Done in a7e5236

**Pull request merged**